TACC/Core-Portal
# Task/FP-1085: Add button for recreating storage systems on onboarding.

## Overview: ##
Add a **Force Create** button to storage systems that mimics the functionality in [force_system_creation.py](https://github.com/TACC/Core-Portal/blob/27c8d956a21f3262e27d53ca9f6fddd771b379df/server/portal/apps/system_creation/management/commands/force_system_creation.py).

## Related Jira tickets: ##

* [FP-1085](https://jira.tacc.utexas.edu/browse/FP-1085)

## Summary of Changes: ##
Added a button on the Admin Onboarding page to trigger recreation of the user's storage system(s).
Moved some code in the `force_system_creation.py` management code to [system_creation/utils.py](https://github.com/TACC/Core-Portal/blob/6cee1bc6ca8bcad7403d104f753a39654c7f2c67/server/portal/apps/system_creation/utils.py) to reduce duplication. And reused that in [onboarding/api/views.py](https://github.com/TACC/Core-Portal/blob/6cee1bc6ca8bcad7403d104f753a39654c7f2c67/server/portal/apps/onboarding/api/views.py) to catch from a request param.

## Testing Steps: ##
1. Get [staff permissions](https://github.com/TACC/Core-Portal/wiki/How-to-Set-Your-User-as-Staff-or-Superuser).
2. Go to `workbench/onboarding/admin`.
3. Have another user without storage? (not sure about this part)
4. Ensure that you see the **Force Create** button.
5. Click and ensure that the storage system is created for the user.

## UI Photos:
<img width="1121" alt="Screen Shot 2022-03-16 at 3 31 46 PM" src="https://user-images.githubusercontent.com/9425579/158685581-5a805312-33b2-4cc3-9720-1647ce56989e.png">

## Notes: ##
I wasn't sure how to go about handling the user's onboarding state for the Storage creation step after force create is triggered. Would we assume that that step is complete?
